src/main/java/application/dao/jdbc/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,174 @@
+package application.dao.jdbc;
+
+import application.dao.OrderDao;
+import application.exceptions.DataProcessingException;
+import application.lib.Dao;
+import application.model.Order;
+import application.model.Product;
+import application.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoJdbcImpl implements OrderDao {
+    @Override
+    public List<Order> getUserOrders(Long userId) {
+        List<Order> orders = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(
+                         "SELECT * FROM orders WHERE user_id = ?")) {
```

Comment:
Let's introduce a separate variable for every similar case

_**String query = "SELECT * FROM orders WHERE user_id = ?";**_